integration/advent2011-day24.t

```diff
@@ -31,9 +31,9 @@ multi sub Slurp($filename) {
 }
 --END--
 
-lives-ok { EVAL $unambiguous ~ 'Slurp("README.md")' },
+lives-ok { EVAL $unambiguous ~ 'Slurp("'~$?FILE~'")' },
   'unambiguous multi - lives';
-throws-like { EVAL $ambiguous   ~ 'Slurp("README.md")' },
+throws-like { EVAL $ambiguous   ~ 'Slurp("'~$?FILE~'")' },
```

Comment:
This line and the one above aren't safe if the user is executing roast from a dir that has a double-quote in its name. Need to use `'Slurp('~$?FILE.perl~')'`; or just `EVAL "$ambiguous Slurp($?FILE.perl())"`